README.md

```diff
@@ -1,9 +1,233 @@
-# winforms-syntax-editor-getting-started
+# WinForms Syntax Editor Getting Started
 
-The EditControl is a powerful text editor control to create interactive code editor applications with its unique feature set. It has many efficient features such as editing, syntax highlighting, text indentation, intellisense, expand or collapse a block of code, and custom language configuration as in Microsoft Visual Studio Editor.
+The [WinForms Syntax Editor](https://www.syncfusion.com/winforms-ui-controls/syntax-editor) is a powerful text editor control to create interactive code editor applications with its unique feature set. It has many efficient features such as editing, syntax highlighting, text indentation, intellisense, expand or collapse a block of code, and custom language configuration as in Microsoft Visual Studio Editor.
 
 ![](Images/SyntaxEditor_Output.png)
 
 For know more about SyntaxEditor : https://www.syncfusion.com/winforms-ui-controls/syntax-editor
 
 SyntaxEditor user guide documentation : https://help.syncfusion.com/windowsforms/syntax-editor/overview
+
+## Overview
```

Comment:
this is the overview content 
![image](https://user-images.githubusercontent.com/61148953/159418829-8cf4069c-523c-43d5-a0e2-a72ab20ee796.png)
don't add separate overview section

Reply:
removed